cfpb/consumerfinance.gov
# Add validation to live streaming fields

Resolves #1446

---

## Changes

* Defaults Live Stream Start Date to Event Start Date if Live Stream Start Date is not defined.
* Requires Live Stream Start Date is not on or after Event End Date if Event End Date is defined.


## How to test this PR

1. Check "Streaming" for all test cases
2. Don't define Live Stream Start Date, save page, ensure Live Stream Start date matches Event Start Date
3. Define event end date and start date after event end date, ensure validation error


## Screenshots
<img width="418" alt="image" src="https://user-images.githubusercontent.com/59629491/155705408-21fa7ece-d6cf-485f-a953-c0f5b2f48258.png">

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
